homework_1/task_3/main.go

```diff
@@ -0,0 +1,101 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/pkg/errors"
+	"io/ioutil"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/sethgrid/pester"
+)
+
+type pokemon struct {
+	Location_area_encounters	string
+	Name						string
+}
+
+type location struct {
+	Location_area				map[string]string
+}
```

Comment:
What's with the formatting?

If you're using GoLand: `Code` -> `Refactor Code` in the menu.
Alternatively, google how to use `go fmt` manually.